databricks-dbutils-scala/src/main/scala/com/databricks/sdk/scala/dbutils/SdkDBUtilsImpl.scala

```diff
@@ -152,3 +154,52 @@ private class SdkSecretsUtils(client: WorkspaceClient) extends SecretUtils with
       SecretScope(scope.getName)
     }
 }
+
+private class SdkJobsUtils extends JobsUtils with NoHelp {
+  override val taskValues: TaskValuesUtils = new SdkTaskValues
+}
+
+private class SdkTaskValues extends TaskValuesUtils with NoHelp {
+  private var commandContext =
+    CommandContext(rootRunId = None, currentRunId = None, jobGroup = None, tags = Map.empty, extraContext = Map.empty)
+
+  /**
+   * Sets a task value on the current task run. This method is a no-op if used outside of the job context.
+   *
+   * @param key
+   *   the task value's key
+   * @param value
+   *   the value to be stored (must be JSON-serializable)
+   */
+  override def set(key: String, value: Any): Unit = throw new NotImplementedError("set is not supported in Scala.")
+
+  /**
+   * Returns the latest task value that belongs to the current job run.
+   *
+   * @param taskKey
+   *   the task key of the task value
+   * @param key
+   *   the key of the task value
+   * @param default
+   *   the value to return when called inside of a job context if the task value does not exist (must not be None)
+   * @param debugValue
+   *   the value to return when called outside of a job context (must not be None)
+   * @return
+   *   the task value (if it exists) when called inside of a job context
+   */
+  override def get(taskKey: String, key: String, default: Option[Any], debugValue: Option[Any]): Any =
+    throw new NotImplementedError("get is not supported in Scala.")
+
+  override def setJson(key: String, value: String): Unit =
+    throw new NotImplementedError("setJson is not supported for local development in Scala.")
```

Comment:
`setJson` should not throw an error, otherwise, code that runs on DBR would throw an error during local development. Instead, we want parity, so we simply swallow it. See the [two-pager](https://docs.google.com/document/d/1yrXXA27GJOB968apgnxnMsWfj0SJJn2cpArE4zzIORU/edit#bookmark=id.j15664rxdtvz) for requirements.

Reply:
Oh, yeah, totally misread this